public/app/features/dashboard/state/initDashboard.test.ts

```diff
@@ -206,8 +205,8 @@ describeInitScenario('Initializing home dashboard', (ctx) => {
   });
 
   it('Should redirect to custom home dashboard', () => {
-    expect(ctx.actions[1].type).toBe(updateLocation.type);
-    expect(ctx.actions[1].payload.path).toBe('/u/123/my-home');
+    const location = locationService.getCurrentLocation();
+    expect(location.pathname).toBe('/u/123/my-home');
```

Comment:
Example how easy this is to text now without any mocking or calling setLocationService